cloud_governance/policy/aws/monitor/cluster_run.py

```diff
@@ -54,6 +54,8 @@ def run_policy_operations(self):
                 else:
                     running_instances = 1
                 creation_date = ''
+                rosa_cluster = True if self.get_tag_name_from_tags(tags=tags, tag_name='red-hat-managed') else False
+                using_graviton = True if '.g' in instance.get('InstanceType') else False
```

Comment:
graviton_families = ["a1", "t4g", "c6g", "m6g", "r6g", "g5g"]
I think we need to have environment variable for it ?

Reply:
Please refer [this](https://github.com/redhat-performance/cloud-governance/pull/852)